FordLabs/PeopleMover
# Add meta.json and cachebusting

## Issue
Resolves #66

## What was done
- [x] Added versioning that allows us to delete caches if the app version it out of date
- [x] Actually did the caching

## How to test
1. Open Firefox (the best browser to tell if you're cached or not)
2. Load the site
3. Under the network tab, in the "transferred" column, see that nothing says "cached"
4. Refresh the site.
5. Under the network tab, see that now a number of items have become cached. 
6. Bonus: check the 'meta.json' row's response and see that it has the latest version

### FOR DEVS
Feature URL: See feature branch

### FOR APPROVAL
Approval URL: See Approval branch
